setup.cfg

```diff
@@ -1,16 +1,22 @@
 [egg_info]
-tag_build = a2
-
-[wheel]
-universal = 1
+# TODO replace
+tag_build = dev5
 
 [aliases]
 test=pytest
 
 [tool:pytest]
-filterwarnings =
-    ignore:Unknown REQUEST_METHOD. '(CONNECT|DELETE|GET|HEAD|OPTIONS|PATCH|POST|PUT|TRACE|CHECKIN|CHECKOUT|COPY|LOCK|MKCOL|MOVE|PROPFIND|PROPPATCH|REPORT|UNCHECKIN|UNLOCK|UPDATE|VERSION-CONTROL)':wsgiref.validate.WSGIWarning
-    ignore:Unknown REQUEST_METHOD. '(FOO|BAR|BREW|SETECASTRONOMY)':wsgiref.validate.WSGIWarning
-    ignore:"@coroutine" decorator is deprecated:DeprecationWarning
-    ignore:Using or importing the ABCs:DeprecationWarning
-    ignore:cannot collect test class 'TestClient':pytest.PytestCollectionWarning
+# filterwarnings =
+#     ignore:Unknown REQUEST_METHOD. '(CONNECT|DELETE|GET|HEAD|OPTIONS|PATCH|POST|PUT|TRACE|CHECKIN|CHECKOUT|COPY|LOCK|MKCOL|MOVE|PROPFIND|PROPPATCH|REPORT|UNCHECKIN|UNLOCK|UPDATE|VERSION-CONTROL)':wsgiref.validate.WSGIWarning
+#     ignore:Unknown REQUEST_METHOD. '(FOO|BAR|BREW|SETECASTRONOMY)':wsgiref.validate.WSGIWarning
+#     ignore:"@coroutine" decorator is deprecated:DeprecationWarning
+#     ignore:Using or importing the ABCs:DeprecationWarning
+#     ignore:cannot collect test class 'TestClient':pytest.PytestCollectionWarning
+
+env =
+    FALCON_ASGI_WRAP_NON_COROUTINES = Y
+    FALCON_TESTING_SESSION = Y
+    PYTHONASYNCIODEBUG = 1
+
+[flake8]
+max-line-length = 100
```

Comment:
This was a mistake in stashing. it will be removed once I revert the `tag_build`